__tests__/app.test.js

```diff
@@ -290,4 +291,95 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(body.msg).toBe('Not found');
       });
   });
+})
+describe("PATCH /api/articles/:article_id", () => {
+  test("status:200, responds with an updated article when votes are incremented", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({
+        inc_votes: 1000
+      })
+      .set("Accept", "application/json")
+      .expect(200)
+      .then(({body}) => {
+        const {article} = body
+        expect(article.votes).toBe(1100);
+        expect(article.article_id).toBe(1);
+        expect(article).hasOwnProperty('title');
+        expect(article).hasOwnProperty('topic');
+        expect(article).hasOwnProperty('author');
+        expect(article).hasOwnProperty('body');
+        expect(article).hasOwnProperty('created_at');
+        expect(article).hasOwnProperty('votes');
+        expect(article).hasOwnProperty('article_img_url');
+      });
+  });
+  test("status:200, responds with an updated article when votes are decremented", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({
+        inc_votes: -100
+      })
+      .set("Accept", "application/json")
+      .expect(200)
+      .then(({body}) => {
+        const {article} = body
+        expect(article.votes).toBe(0);
+        expect(article.article_id).toBe(1);
+        expect(article).hasOwnProperty('title');
+        expect(article).hasOwnProperty('topic');
+        expect(article).hasOwnProperty('author');
+        expect(article).hasOwnProperty('body');
+        expect(article).hasOwnProperty('created_at');
+        expect(article).hasOwnProperty('votes');
+        expect(article).hasOwnProperty('article_img_url');
+      });
+  });
+  test("status:404, sends an appropriate status and error message when patching a valid but non-existent article_id", () => {
+    return request(app)
+      .patch("/api/articles/22222")
+      .send({
+        inc_votes: -100
+      })
+      .set("Accept", "application/json")
+      .expect(404)
+      .then(({body}) => {
+        expect(body.msg).toBe('article does not exist');
+      });
+  });
+  test("status:400, sends an appropriate status and error message when patching an invalid article_id", () => {
+    return request(app)
+    .patch("/api/articles/not_an_article_id")
+    .send({
+      inc_votes: -100
+    })
+    .set("Accept", "application/json")
+      .expect(400)
+      .then(({body}) => {
+        expect(body.msg).toBe('Bad request');
+      });
+  });
+  test("status:400, sends an appropriate status and error message when patching without all properties of new vote request body", () => {
+    return request(app)
```

Comment:
If no 'inc_votes' is sent, it may be better just to return a 200 and return the original, unaltered article.